src/widgets/vitals/vital-record.component.tsx

```diff
@@ -28,7 +30,16 @@ export default function VitalRecord(props: VitalRecordProps) {
   return (
     <>
       {vitalSigns && (
-        <SummaryCard name="Vital" styles={{ width: "100%" }}>
+        <SummaryCard
+          name="Vital"
```

Comment:
Can we translate `Vital`? Sorry I know it was not part of your PR - just having a dedicated eye on localization right now.